src/btree/bt_slvg.c

```diff
@@ -512,26 +511,6 @@ err:	__wt_free(session, trk->trk_addr);
 }
 
 /*
- * __slvg_trk_split --
- *	Split a tracked chunk.
- */
-static int
-__slvg_trk_split(WT_SESSION_IMPL *session, WT_TRACK *orig, WT_TRACK **newp)
```

Comment:
Hmm, I'm more a fan of renaming the function to make it clear we allocated, like `__slvg_trk_alloc` or `__slvg_trk_split_alloc` than duplicating the code in two places.  Otherwise these LGTM.